pyproject.toml

```
@@ -54,6 +54,8 @@ generate_sw_vel_tile = "opera_disp_tms.generate_sw_vel_tile:main"
 create_tile_map = "opera_disp_tms.create_tile_map:main"
 
 [tool.pytest.ini_options]
+addopts = '-ra -q -m "not integration"'
```

Is there a reason for `-q` here? We don't use that anywhere else, and I'd prefer having the extra info.
```suggestion
addopts = '-ra -m "not integration"'
```

---

No strong preference there, I just tend to lean towards less verbose in general.